README.md

```diff
@@ -1,7 +1,18 @@
 # Source Repository for graphql.org
 
-This repository contains the source code of https://graphql.org website.
+This repository contains the source code of [GrapQL](https://graphql.org)
+> [Invite link to slack channel](https://graphql-slack.herokuapp.com/)
 
-## Contributing
+**GraphQL 🎉** is a query language for APIs and a runtime for fulfilling those queries with your existing data.
 
-Check out our [contributing guide](./CONTRIBUTING.md) for detailed instructions for how to make changes to graphql.org 🎉
\ No newline at end of file
+**GraphQL 🎉** provides a complete and understandable description of the data in your API, gives clients the power to ask for exactly what they need and nothing more, makes it easier to evolve APIs over time, and enables powerful developer tools.
+
+## Documentation
+- [Site](https://graphql.org/)
+- [User guide](https://graphql.org/code/)
+- [API documentation](https://graphql.org/learn/)
```

Comment:
`API` isn't quite right here, as there's a separate site for the [spec documentation](https://spec.graphql.org/) (which could also be worth noting in this section). `Reference` might be a more appropriate word choice.

```suggestion
- [Reference documentation](https://graphql.org/learn/)
```

I'd also move this point above the Code page as it's the most prominent part of the site.